src/test/operators/table_scan_test.cpp

```diff
@@ -1020,6 +1020,13 @@ TEST_P(OperatorsTableScanTest, GetImpl) {
     EXPECT_EQ(impl->right_value, AllTypeVariant{4.099999904632568359375f});
   }
 
+  {
+    const auto abstract_impl = TableScan{get_int_float_op(), between_exclusive_(column_a, column_b, 3.1)}.create_impl();
```

Comment:
Would it make sense to add a swapped version as well?
E.g.: 

```suggestion
     const auto abstract_impl = TableScan{get_int_float_op(), between_exclusive_(column_a, 3.1, column_b)}.create_impl();
```

Reply:
Doesn't hurt I guess